k8s.io/configmap-www-golang.yaml

```diff
@@ -4,6 +4,26 @@ metadata:
   name: www-golang
 data:
   # Adding new entries here will make them appear as files in the deployment.
+  contrib.html: |
```

Comment:
Should it be `cloud-provider-aws.html` instead of `contrib.html`?

Reply:
yes

Reply:
Done!